app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+
+  validates :user_id, presence: true
```

Comment:
Hi, we are missing validations for `body` and `title` of our post. Let us add them